deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -594,329 +532,597 @@ spec:
+                  value: "true"
+                - name: ROOK_ENABLE_FLEX_DRIVER
+                  value: "false"
+                - name: ROOK_ENABLE_DISCOVERY_DAEMON
+                  value: "false"
+                - name: ROOK_DISABLE_DEVICE_HOTPLUG
+                  value: "true"
+                - name: AGENT_TOLERATION
+                  value: NoSchedule
+                - name: AGENT_TOLERATION_KEY
+                  value: node.ocs.openshift.io/storage=true
+                - name: AGENT_NODE_AFFINITY
+                  value: cluster.ocs.openshift.io/openshift-storage=
+                - name: DISCOVER_TOLERATION
+                  value: NoSchedule
+                - name: DISCOVER_TOLERATION_KEY
+                  value: node.ocs.openshift.io/storage=true
+                - name: DISCOVER_AGENT_NODE_AFFINITY
+                  value: cluster.ocs.openshift.io/openshift-storage=
+                - name: NODE_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: spec.nodeName
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                image: rook/ceph:master
+                name: rook-ceph-operator
+                resources: {}
+                volumeMounts:
+                - mountPath: /var/lib/rook
+                  name: rook-config
+                - mountPath: /etc/ceph
+                  name: default-config-dir
+              serviceAccountName: rook-ceph-system
+              volumes:
+              - emptyDir: {}
+                name: rook-config
+              - emptyDir: {}
+                name: default-config-dir
+      - name: noobaa-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              noobaa-operator: deployment
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app: noobaa
+                noobaa-operator: deployment
+            spec:
+              containers:
+              - env:
+                - name: OPERATOR_NAME
+                  value: noobaa-operator
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                image: noobaa/noobaa-operator:1.0.2
+                imagePullPolicy: IfNotPresent
+                name: noobaa-operator
+                resources:
+                  limits:
+                    cpu: 250m
+                    memory: 256Mi
+              serviceAccountName: noobaa-operator
+      - name: ocs-operator
```

Comment:
The deployment of ocs-operator has been added a second time here. Didn't notice it until I tried to set an alternate image manually, and had two ocs-operator pods running.